### PR TITLE
Update WaitOnAdress / WakeByAddressSignle / WakeByAddressAll to refer to the correct dll

### PR DESCRIPTION
**Pull request by AlexGuteniev**
The page [Windows 8 API Sets](https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-8-api-sets) lists `api-ms-win-core-synch-l1-2-0.dll`.
Debugging shows it actually maps to `KernelBase.dll`

Attempts to get these functions from either `KernelBase.dll` or `api-ms-win-core-synch-l1-2-0.dll` work. Attempts to get them from `Kernel32.dll` fail.

So I'm really not sure which of `KernelBase.dll` or `api-ms-win-core-synch-l1-2-0.dll` is correct, but sure not `Kerne32.dll`

### CODE REVIEW

**Comment by PRMerger17** [comment]
@AlexGuteniev : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]
Thanks @AlexGuteniev. I confirmed with a feature team dev that your suggested update is correct.

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Comment by Karl-Bridge-Microsoft** [comment]
Hi @AlexGuteniev, can you resolve the merge conflicts and resubmit? Thanks!

**Comment by PRMerger18** [comment]
@AlexGuteniev : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by AlexGuteniev** [comment]
I was not able to merge it in a clean way, I reverted and re-applied changes.

File: sdk-api-src/content/synchapi/nf-synchapi-wakebyaddressall.md [hunk 1]
```diff
@@ -26,7 +27,7 @@ req.namespace:
 req.assembly: 
 req.type-library: 
 req.lib: Synchronization.lib
-req.dll: Kernel32.dll
+req.dll: API-MS-Win-Core-Synch-l1-2-0.dll.dll
```
Comment (Lastique) [inline sdk-api-src/content/synchapi/nf-synchapi-wakebyaddressall.md]: .dll.dll? Same below.

**Comment by PRMerger17** [comment]
@AlexGuteniev : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/synchapi/nf-synchapi-wakebyaddressall.md (hunk 1 above)
Comment (AlexGuteniev) [reply sdk-api-src/content/synchapi/nf-synchapi-wakebyaddressall.md]: Good catch! fixed.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Pull request merged**